t/script/cover.t

```diff
@@ -1,14 +1,18 @@
 use strict;
 use warnings;
+
 use lib 't/lib';
 
 use MetaCPAN::Script::Cover  ();
-use MetaCPAN::Script::Runner ();
+use MetaCPAN::Server::Config ();
 use MetaCPAN::Util           qw( checkout_root );
+use Test::More import => [qw( done_testing ok )];
```

Comment:
This is a duplicate use, but also tries to enumerate the imports to use. That's a good idea in general, but I don't like it for Test::More, and we have it excluded from the import checks.

Reply:
Yeah, not sure how this one happened.